workflows/ogg_to_mp3.py

```diff
@@ -0,0 +1,50 @@
+import os
+from multiprocessing import Pool
+from pathlib import Path
+
+import luigi
+from google.cloud import storage
+from luigi.parameter import ParameterVisibility
+from pydub import AudioSegment
+
+
+class OggToMP3(luigi.Task):
+    input_path = luigi.Parameter()
+    output_path = luigi.Parameter()
+    parallelism = luigi.IntParameter(default=os.cpu_count(), significant=False)
+
+    def output(self):
+        return luigi.LocalTarget(self.output_path)
+
+    def list_images(self):
+        # storage_client = storage.Client()
+
+        # Note: Client.list_blobs requires at least package version 1.17.0.
+        # images = storage_client.list_blobs(self.input_path)
+        images = os.listdir(self.input_path)
```

Comment:
Unless this lists files recursively, I think this will only list species names if you pass it the `train_audio` or `birdclef-2022` root. I like use `pathlib.Path.glob` for this kind of thing e.g. `Path(path).glob("**/*.ogg")`.

I'm conflicted on how we should organize this. One approach that's appealing is to have a separate task that pulls down from GCS species at a time, converts species at a time, and then uploads species at a time.

```mermaid
graph LR
  pull --> convert
  convert --> push
```

This would be captured in a larger meta-task that can parse through a species list and connect all the pieces together.

On the other-hand, there's appeal in doing it in a simple loop. If we lose progress half-way, it'll probably be hard to make up progress here.